package.json

```diff
@@ -6,12 +6,12 @@
   "description": "SAP Fundamental Styles, implemented in Angular",
   "scripts": {
     "build": "ng build",
-    "build-all": "ng build core && ng build platform && ng build docs",
-    "build-docs": "npm run compile-typedoc && npm run copy-docs && ng build --prod --base-href=\"/fundamental-ngx/\" ",
-    "build-docs-netlify": "npm run build-all && npm run compile-typedoc && npm run copy-docs && ng build --prod --base-href=\"/\" ",
-    "build-deploy-library": "ng build core && ng build platform && npm run build:schematics && npm run sync-versions && cp README.md dist/libs/core/README.md",
+    "build-all": "ng build core && ng build platform && ng build docs && npm run build-docs",
+    "build-docs": "npm run compile-typedoc-all && npm run copy-docs && ng build --prod --base-href=\"/fundamental-ngx/\" ",
+    "build-docs-netlify": "npm run build-all && npm run compile-typedoc-all && npm run copy-docs && ng build --prod --base-href=\"/\" ",
+    "build-deploy-library": "ng build core && ng build platform && npm run build:schematics && npm run sync-versions && cp README.md dist/libs/core/README.md && cp PLATFORM_README.md dist/libs/platform/PLATFORM_README.md",
     "build-pack-library": "ng build core && ng build platform && npm run build:schematics && npm run sync-versions && cd dist/libs/core/ && npm pack && cd ../platform && npm pack && cd ../..",
-    "copy-docs": "cp README.md apps/docs/src/assets/README.md && cp NEW_COMPONENT.md apps/docs/src/assets/NEW_COMPONENT.md && npm run copy-app-docs",
+    "copy-docs": "cp README.md apps/docs/src/assets/README.md && cp NEW_COMPONENT.md apps/docs/src/assets/NEW_COMPONENT.md && cp PLATFORM_README.md apps/docs/src/assets/PLATFORM_README.md && cp NEW_PLATFORM_COMPONENT.md apps/docs/src/assets/NEW_PLATFORM_COMPONENT.md && npm run copy-app-docs",
```

Comment:
Put `cp platform_readme.md apps/docs/src/assets/platform-readme.md` to `copy-app-docs`

Reply:
Done